algviz/docs/list/list.md

```diff
@@ -0,0 +1,13 @@
+# Lists
+
+To generate an SVG for a list simply call the constructor
+
+```python
+from algviz.list.listvisualizer import ListVisualizer
+
+ListVisualizer("listVisualization.svg", [1,2,3], (150, 50))
```

Comment:
If you're glazing over the fact that we're "simply" passing a mysterious parameter `(150, 50)` to this function, that's a sign that the interface doesn't quite make sense.
Also, if we're supposed to use `ListVisualizer` as a function and not as a constructor, it should be lowercased, as in `list_visualizer` (or better, `draw_list(...)`).